src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const newNotification = document.createElement('div');
+
+  newNotification.setAttribute('class', `notification ${type}`);
+
+  const titleElement = document.createElement('h2');
+
+  titleElement.textContent = title;
+  newNotification.appendChild(titleElement);
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.textContent = description;
+  newNotification.appendChild(descriptionElement);
+
+  newNotification.style.top = `${posTop}px`;
+  newNotification.style.right = `${posRight}px`;
+
+  document.body.appendChild(newNotification);
+
+  setTimeout(() => {
+    newNotification.remove();
+  }, 2000);
```

Comment:
The timeout duration for removing the notification is set to 2000 milliseconds (2 seconds). Depending on the use case, this might be too short for users to read and understand the message. Consider increasing the timeout duration or making it configurable.